pages/index.html

```diff
@@ -1,39 +1,167 @@
 ---
 permalink: /
-layout: styleguide
+layout: homepage
 title: U.S. Federal Web Design Standards
 ---
 
-<h1>About this Guide</h1> 
+<a class="skipnav" href="#main-content">Skip main navigation</a>
 
-<p>Welcome to the alpha site for the U.S. Federal Web Design Standards project.</p> 
+<header role="banner">
+  
+  <div class="usa-disclaimer">
+    <div class="usa-grid">
+      <span class="usa-disclaimer-official">
+        <img class="usa-flag_icon" alt="US flag signifying that this is a United States Federal Government website" src="{{ site.baseurl }}/assets/img/us_flag_small.png">
+        An official website of the United States Government
+      </span>
+      <span class="usa-disclaimer-stage">This site is currently in alpha. <a href="#">Learn more.</a></span>
+    </div>
+  </div>
+  
+  <section class="usa-banner">
+    <div class="usa-grid">
+      <nav>
+        <a class="usa-banner-link-top" href="https://github.com/18F/usfwds">View on GitHub</a>
+      </nav>
+      <div class="usa-banner-content" id="main-content">
+        <h1>U.S. Federal Web Design Standards</h1>
+        <h2 class="usa-lead">Open source UI components, visual style guide and content style guide for US federal government websites.</h2>
+      </div>
+        <a class="usa-button usa-button-big usa-button-secondary" href="{{ site.baseurl }}/getting-started">View the standards</a>
+        <div class="usa-button-block">
+          <a class="usa-button usa-button-big usa-button-outline-inverse">Download the components</a>
+          <p class="usa-text-small">Download a zip file with the assets of this library</p>
+        </div>
+    </div>
+  </section>
+  
+</header>
 
-<p>This web design standards is intended to provide reference for designers, developers, as well as anyone else interested in the design and frontend principles that make up the building blocks of federal government websites.</p>
+<section class="usa-section usa-section-alt">
+  <div class="usa-grid">
+    <div class="usa-intro">
+      <h2>Tools for creating beautiful online experiences for the American people.</h2>
+      <p class="intro-text">Built and maintained by a team of US Digital Service + 18F designers and developers, this resource is built on the highest standards of 508 compliance, reuses best practices of existing style libraries and modern web design to guide us in creating beautiful and easy-to-use online experiences for the American people.</p>
+    </div>
+    <div class="usa-width-one-half">
+      <a href="#">
+        <div class="usa-img-secondary">
+          <img src="{{ site.baseurl }}/assets/img/home/homepage_illustrations_ui_components.png" alt="UI Components">
+        </div>
+        <h3>UI Components</h3>
+      </a>
+      <p>Common web interactions (buttons, forms, navigation, etc.) with reusable and downloadable code</p>
+    </div>
+    <div class="usa-width-one-half usa-end-row">
```

Comment:
I see what's going on here, a problem with our grid or in general, how the neat grid works.

Technically if you have two one halfs next to each other, the second one should not have the margin right, meaning it won't break to the next line. This isn't happening here because you have `<div class="usa-intro">` first under `usa-grid`. This causes the first `one-half` to have the margin-right 0. 

One way to fix would be to move `usa-intro` out of this `usa-grid` div and into it's own above. Although I'm curious to hear your general thoughts on this.

We do have to be careful about `usa-end-row` because if we want to do interesting things with the grid at various breakpoints, it could break that functionality because it's forcing where the grid breaks.